cfgov/cfgov/settings/local.py

```diff
@@ -40,7 +40,7 @@
 if os.environ.get('ENABLE_DEBUG_TOOLBAR'):
     INSTALLED_APPS += ('debug_toolbar',)
 
-    INTERNAL_IPS = ('127.0.0.1', '10.0.2.2')
+    INTERNAL_IPS = (os.environ.get('INTERNAL_IP', '127.0.0.1'), )
```

Comment:
Not being super familiar with the Docker setup, if you were to provide an `INTERNAL_IP` variable here, and thus `INTERNAL_IPS` would get set to **only** that variable's value (not including `'127.0.0.1'`), would it break any functionality when accessed via localhost from within the Python container?